submissions/olehmaksymych/html_popup/styl.css

```diff
@@ -0,0 +1,182 @@
+
+:root {
+  --green: #27ae60;
+  --black: #192a56;
+  --light-color: #666;
+  --picture-size: 30px;
+  --border-radious: 3px;
+}
+
+* {
+  margin: 0; padding: 0;
+  box-sizing:border-box;
+  text-decoration: none;
+  outline: none; 
+  border:none;
+  transition: all .2s linear; 
+}
+
+
+
+.header_menu  a.navbar.active,
+.header_menu a.navbar:hover{
```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css), use classnames instead